### PR TITLE
Paperscript es2015

### PR DESCRIPTION
**Pull request by zekenie**
Worked with @roneesh to update Acorn and support ES2015.

We got thrown off by a weird parsing issue, but removing a regular expression seems to have resolved it. We don't quite understand what it was doing. 

Tests are passing. Tried taking the built files and moving them into the sketch editor. A bunch of existing projects & examples worked. Also tried ES6 stuff which worked fine.

### CODE REVIEW

File: src/core/PaperScript.js [hunk 1]
```diff
@@ -195,20 +195,16 @@ Base.exports.PaperScript = function() {
...
                         operator = node.operator;
-                    replaceCode(node, '__$__(' + left + ','
-                            // To preserve line-breaks, get the code in between
-                            // left & right, and replace the occurrence of the
-                            // operator with its string counterpart:
-                            + between.replace(new RegExp('\\' + operator),
-                                '"' + operator + '"')
-                            + ', ' + right + ')');
+                        var string = 
+                             '__$__(' + left+ ',' + '"' + operator + '"' + ', ' + right + ')';
```
Comment (burnpanck) [inline src/core/PaperScript.js]: One could restore line-number correctness by simply counting the number of linebreaks in `between` and adding them before or after the operator: `'\n'.repeat((between.match(/\n/g) || []).length)`

**Comment by mpurgatori** [comment]
Currently making a project with paper and would be great to get this es6 functionality. Get on it Paper!

**Comment by lehni** [comment]
@mpurgatori we do our best and have other work that pay the bills. You can always help out by donating to speed up the handling of features that are important to you. But giving orders here doesn't really help.

**Comment by mpurgatori** [comment]
@lehni  I apologize, the comment was really just tongue in cheek. I love the library and I will be sure to donate to the project as soon as I am employed.

**Comment by lehni** [comment]
@mpurgatori hard to understand irony in comments sometimes, but good to hear you like the library! I'll look into this issue soon.

**Comment by lehni** [comment]
So I have addressed this in #1183. I will write instructions as to how to use an external version Acorn.

**Pull request closed without merging**